cmd/context.go

```diff
@@ -0,0 +1,150 @@
+		if wasCurrent {
+			printer.PrintCurrentContext("")
+		}
+	},
+}
+
+var getContextCmd = &cobra.Command{
+	Use:   "get",
+	Args:  cobra.RangeArgs(0, 1),
+	Short: "Get context, current context by default",
+	Long:  "Get context, current context by default",
+	Example: `corectl context get
+corectl context get local-engine`,
+
+	Run: func(ccmd *cobra.Command, args []string) {
+		handler := internal.NewContextHandler()
+		var name string
+		if len(args) == 1 {
+			name = args[0]
+		} else {
+			name = handler.Current
+		}
+		context := handler.Get(name)
+		printer.PrintContext(name, context)
+	},
+}
+
+var updateContextCmd = withLocalFlags(&cobra.Command{
+	Use:   "update",
+	Args:  cobra.RangeArgs(0, 1),
+	Short: "Update context, current context by default",
+	Long:  "Update context, current context by default",
+	Example: `corectl context update
+corectl context update local-engine`,
+
+	Run: func(ccmd *cobra.Command, args []string) {
+		var name string
+		if len(args) == 1 {
+			name = args[0]
+		}
+		internal.UpdateContext(name, viper.GetString("comment"))
+	},
+}, "comment")
+
+var listContextsCmd = &cobra.Command{
+	Use:     "ls",
+	Args:    cobra.ExactArgs(0),
+	Short:   "List all contexts",
+	Long:    "List all contexts",
+	Example: "corectl context ls",
+
+	Run: func(ccmd *cobra.Command, args []string) {
+		handler := internal.NewContextHandler()
+		printer.PrintContexts(handler, viper.GetBool("bash"))
+	},
+}
+
+var setContextCmd = &cobra.Command{
+	Use:     "set",
+	Args:    cobra.ExactArgs(1),
+	Short:   "Set a current context",
+	Long:    "Set a current context",
+	Example: "corectl context set local-engine",
+
+	Run: func(ccmd *cobra.Command, args []string) {
+		name := internal.SetCurrentContext(args[0])
+		printer.PrintCurrentContext(name)
+	},
+}
+
+var unsetContextCmd = &cobra.Command{
+	Use:     "unset",
+	Args:    cobra.ExactArgs(0),
+	Short:   "Unset current context",
+	Long:    "Unset current context",
+	Example: "corectl context unset",
+
+	Run: func(ccmd *cobra.Command, args []string) {
+		previousContext := internal.UnsetCurrentContext()
+		if previousContext != "" {
+			printer.PrintCurrentContext("")
+		}
+	},
+}
+
+var contextCmd = &cobra.Command{
+	Use:   "context",
+	Short: "Create, update and use contexts",
+	Long: `Create, update and use contexts
+
+Contexts store connection information such as engine url, certificates and headers,
+similar to a config. The main difference between contexts and configs is that they
+can be used globally. Use the context subcommands to configure contexts which
+facilitate app development in environments where certificates and headers are needed.
+
+The current context is the one that is being used. You can use "context get" to
+display the contents of the current context and switch context with "context set"
+or unset the current context with "context unset".
+
+Note that contexts have the lowest precedence. This means that a e.g. an --engine flag
```

Comment:
Extra "a" in "This means that a e.g. an --engine flag"

Reply:
Fixed